test/parallel/test-http2-server-shutdown-redundant.js

```diff
@@ -0,0 +1,28 @@
+// Flags: --expose-http2
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const assert = require('assert');
+const http2 = require('http2');
+
+const server = http2.createServer();
+
+// Test blank return when a stream.session.shutdown is called twice
+server.on('stream', common.mustCall((stream) => {
+  stream.session.shutdown(common.mustCall(() => {
```

Comment:
I would also run another `stream.session.shutdown(common.mustNotCall())` outside of the callback. If there are two calls in a row, it shouldn't throw or execute a callback.

Reply:
Added!